dockerfile/cosmos/local.Dockerfile

```diff
@@ -8,18 +8,16 @@ ARG BUILDARCH
 ARG GITHUB_ORGANIZATION
 ARG REPO_HOST
 ARG GITHUB_REPO
+ARG WASMVM_VERSION
 
 WORKDIR /go/src/${REPO_HOST}/${GITHUB_ORGANIZATION}/${GITHUB_REPO}
 
-# Download dependencies and CosmWasm libwasmvm if found.
-ADD go.mod go.sum ./
```

Comment:
why was this and L22 removed?  We should be able to cache deps still, but it could be done as a phase after the libwasmvm download since that no longer depends on the go.mod being present.

Reply:
Pulling those files does not work when there is a custom build directory